roseau/load_flow/network/electrical_network.py

```diff
@@ -508,11 +496,11 @@ def buses_voltages(self, as_magnitude_angle: bool = False) -> pd.DataFrame:
             Name: voltage, dtype: complex128
         """
         voltages_dict = {"bus_id": [], "phase": [], "voltage": []}
-        phases = {3: ["ab", "bc", "ca"], 4: ["an", "bn", "cn"]}
-        phases_dtype = pd.CategoricalDtype(phases[4] + phases[3], ordered=True)
+        phases = {False: ["ab", "bc", "ca"], True: ["an", "bn", "cn"]}
+        phases_dtype = pd.CategoricalDtype(phases[True] + phases[False], ordered=True)
         for bus_id, bus in self.buses.items():
             voltages = bus.voltages.m_as("V")
-            for voltage, phase in zip(voltages, phases[bus.n]):
+            for voltage, phase in zip(voltages, phases["n" in bus.phases]):
```

Comment:
Warning here. The order is not always "ab", "bc" and "ca". A bus may just have "bc" or "ca"